account_statement_import_move_line/tests/test_account_bank_statement_import_move_line.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0
+
+from odoo import fields
+from odoo.tests import TransactionCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TestAccountBankStatementImportMoveLine(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.partner = cls.env["res.partner"].create({"name": "Test Partner 2"})
+        cls.journal_bank = cls.env["account.journal"].create(
+            {"name": "Test Journal Bank", "type": "bank", "code": "TJB0"}
+        )
+        cls.invoice = cls.env["account.move"].create(
+            {
+                "name": "Test Invoice 3",
+                "partner_id": cls.partner.id,
+                "move_type": "out_invoice",
+                "invoice_line_ids": [
+                    (
+                        0,
+                        0,
+                        {
+                            "name": "Test line",
+                            "quantity": 1.0,
+                            "price_unit": 100.00,
+                        },
+                    )
+                ],
+            }
+        )
```

Comment:
```suggestion
from odoo.tests import tagged
from odoo.addons.account.tests.common import AccountTestInvoicingCommon


@tagged("post_install", "-at_install")
class TestAccountBankStatementImportMoveLine(AccountTestInvoicingCommon):
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.partner = cls.env["res.partner"].create({"name": "Test Partner 2"})
        cls.journal_bank = cls.company_data["default_journal_bank"]
        cls.invoice = cls.init_invoice("out_invoice", partner=cls.partner, amounts=[100])        
```

I think it could be simplified to something like this.